### ISSUE TITLE
need permission to buy/DCA and a Button

### ISSUE THREAD

**Issue by chetubc** [taniman/profit-trailer-enhancements#170]
1- there will be a button to set on/off the permission for PT to ask the user before doing a buy/DCA pair
2- during the time of Buy/DCA if the above permission is set to ask, then PT will ask the user which pair to execute when there are more DCA'd pairs being trialed.
**Description**:
many a time there will be more than one DCA'ed pairs.
suppose there will be more than one pair which will be true-trailing(based on settings)
but there will not be sufficient balance to buy(DCA) all of them. 
I have noticed PT buying totally irreverent pair at some times and I do not think there is a module to decide which one pair will be bought first whenever there is a little balance in our account which might be due to either selling of another pair for profits or adding of the balance.
**How it is going to help me:**
Users when ever see more than one coin is getting true-trailing, they can go check the market and analyze buying which pair would be more profitable at that very moment. 
so, Basically, PT will start showing a popup or a button against the trailing pairs to set priority.
**example scenario:** 
suppose the account balance is enough for only one DCA value pair but more than one pairs are trailing(may be more than one - but balance would be lesser than needed for all trailing DCAs) 
there is this coin which I have more levels of DCA and lesser levels as well. but through this option I can always set it to ask permission whenever I am available, and then set coins only to be considered in case more coins are being trailed.

**Comment by chetubc** [comment]
the same can be extended during our first buy as well when there are more coins under trailing and there is our fav coin being trialed along with some other not so good ones. in this case we can ask PT to wait for our fav coin to get a valid buy or just go and buy our fav coin first.

cheers!

**Comment by chetubc** [comment]
I know this permission window cannot be there for ever for user to make the decision but if PT found inactive for like more than 60mins on GUI then safely asking fr this permission can be omitted.

**Comment by clownfish44** [comment]
Already requested in #7
